.github/workflows/build-test.yml

```diff
@@ -0,0 +1,45 @@
+name: Snyk Vulnerability Scan
+
+on:
+  push:
+    branches:
+      - main 
+  pull_request:
+    branches:
+      - main  
+
+jobs:
```

Comment:
We need to add some jobs in here. Please add a job to do a yarn build:electron step. this is the script we have that will build all of our app. If this doesn't succeed then we don't want to continue with a snyk scan.

Reply:
added yarn build:electron under jobs unfortunately snyk is not working